pliers/extractors/text.py

```diff
@@ -342,44 +341,44 @@ def __init__(self, model='en_core_web_sm', features=None):
         verify_dependencies(['spacy'])
         try: 
```

Comment:
If the `try` block fails, `self.model` and `self.features` won't exist, so extraction will fail when the user calls `transform()`, even if the download process succeeded.